package.json

```diff
@@ -34,11 +34,13 @@
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.4",
     "@svgr/webpack": "^8.1.0",
     "add-asset-html-webpack-plugin": "^6.0.0",
+    "axios": "^1.3.6",
```

Comment:
Do we really need axios? `fetch` is pretty nice, and widely available in 2025. Can we just use that if we bump `engines.node` to something that's actually currently supported?